ReactAndroid/src/main/java/com/facebook/react/devsupport/DevSupportManager.java

```diff
@@ -367,10 +367,18 @@ public void onReactInstanceDestroyed(ReactContext reactContext) {
   }
 
   public String getSourceMapUrl() {
+    if (mJSAppBundleName == null) {
+      return "";
+    }
+
     return mDevServerHelper.getSourceMapUrl(Assertions.assertNotNull(mJSAppBundleName));
   }
 
   public String getSourceUrl() {
+    if (mJSAppBundleName == null) {
+      return "";
+    }
+
```

Comment:
This is going to silently break source maps in redboxes when the developer doesn't provide the JSAppBundleName. Please file an issue to revert this change, go back to enforcing passing the app bundle name, and instead check if the packager is running during reload.